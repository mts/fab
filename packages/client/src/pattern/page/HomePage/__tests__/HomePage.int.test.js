import React from 'react'
import { asyncRender, defaultRender } from './HomePage.int.render'

jest.mock('react-router-dom', () => ({ Link: () => <div>some link</div> }))

describe('<HomePage />', () => {
  afterEach(() => jest.clearAllMocks())

  describe('Snaphot', () => {
    test('must match asyncRender', () => {
      expect(global.renderToJSON(asyncRender)).toMatchSnapshot()
    })

    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
