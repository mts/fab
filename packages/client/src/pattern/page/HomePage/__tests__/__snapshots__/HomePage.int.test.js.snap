// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> Snaphot must match asyncRender 1`] = `null`;

exports[`<HomePage /> Snaphot must match defaultRender 1`] = `
<div
  className=""
>
  <div
    className="homeTemplate"
  >
    HomeTemplate under construction
  </div>
</div>
`;
